modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -126,6 +129,76 @@ class ONNXImporter
     void handleNode(const opencv_onnx::NodeProto& node_proto);
 };
 
+std::set<String> ONNXImporter::supportedTypes = {
```

Comment:
Lets minimize global mandatory variables initialization. Move them to functions with initialization on-demand.

```
/*static*/
const std::set<String>& ONNXImporter::getSupportedTypes()
{
    static const std::set<String> g_types = {
        ...
    };
    return g_types;
}
```